### ISSUE TITLE
[Windows compatibility issue ]python setup.py develop error occurred

### ISSUE THREAD

**Issue by jayahn77777** [vita-epfl/UniTraj#25]
Windows 10
nvcc --version
nvcc: NVIDIA (R) Cuda compiler driver
Copyright (c) 2005-2022 NVIDIA Corporation
Built on Wed_Sep_21_10:41:10_Pacific_Daylight_Time_2022
Cuda compilation tools, release 11.8, V11.8.89
Build cuda_11.8.r11.8/compiler.31833905_0
ninja --version
1.12.1

I followed below instruction.

Create a new conda environment
conda create -n unitraj python=3.9
conda activate unitraj
Install ScenarioNet: https://scenarionet.readthedocs.io/en/latest/install.html

Install Unitraj:

git clone https://github.com/vita-epfl/UniTraj.git
cd unitraj
pip install -r requirements.txt
python setup.py develop

python setup.py develop --> this command shows below error.

**(unitraj) PS D:\Coding\Anaconda\UniTraj_01\UniTraj> python setup.py develop**
D:\Coding\Anaconda\UniTraj_01\UniTraj\unitraj\models\mtr\ops\src\knn.cpp
running develop
C:\Users\ahn_j\.conda\envs\unitraj\lib\site-packages\setuptools\command\develop.py:41: EasyInstallDeprecationWarning: easy_install command is deprecated.
!!

        ********************************************************************************
        Please avoid running ``setup.py`` and ``easy_install``.
        Instead, use pypa/build, pypa/installer or other
        standards-based tools.

        See https://github.com/pypa/setuptools/issues/917 for details.
        ********************************************************************************

!!
  easy_install.initialize_options(self)
C:\Users\ahn_j\.conda\envs\unitraj\lib\site-packages\setuptools\_distutils\cmd.py:66: SetuptoolsDeprecationWarning: setup.py install is deprecated.
!!

        ********************************************************************************
        Please avoid running ``setup.py`` directly.
        Instead, use pypa/build, pypa/installer or other
        standards-based tools.

        See https://blog.ganssle.io/articles/2021/10/setup-py-deprecated.html for details.
        ********************************************************************************

!!
  self.initialize_options()
running egg_info
writing UniTraj.egg-info\PKG-INFO
writing dependency_links to UniTraj.egg-info\dependency_links.txt
writing top-level names to UniTraj.egg-info\top_level.txt
reading manifest file 'UniTraj.egg-info\SOURCES.txt'
adding license file 'LICENSE'
writing manifest file 'UniTraj.egg-info\SOURCES.txt'
running build_ext
error: [WinError 2] The system cannot find the file specified.

I don't know what I missed......

**Comment by VVeiCao** [comment]
Try
`pip install setuptools==58.2.0`

**Comment by jayahn77777** [comment]
Thanks for help

I tried the command "pip install setuptools==58.2.0"

After the command I got below error.
(unitraj) PS D:\Coding\Anaconda\UniTraj_01\UniTraj> python setup.py develop
running develop
running egg_info
writing UniTraj.egg-info\PKG-INFO
writing dependency_links to UniTraj.egg-info\dependency_links.txt
writing top-level names to UniTraj.egg-info\top_level.txt
reading manifest file 'UniTraj.egg-info\SOURCES.txt'
adding license file 'LICENSE'
writing manifest file 'UniTraj.egg-info\SOURCES.txt'
running build_ext
error: [WinError 2] The system cannot find the file specified.

After the command, warnings were deleted.
But, I still have same error.....
